### DISCUSSION TITLE
how to prevent 'RuntimeError: dictionary changed size during iteration' ?

### DISCUSSION THREAD

**Discussion by jskcnsl** [agronholm/apscheduler#867] [Q&A]
https://github.com/agronholm/apscheduler/issues/753

I'm having the same problem in above link, is there any way to avoid this ? 
Like change arguments of apscheduler job from object (which may be changed) to int, or something else ?

And I found this error will block all jobs before I restart it.

**Comment by agronholm** [comment]
Please post instructions for how to reproduce the issue.

**Reply by jskcnsl** [reply]
Sorry, in fact I'm not sure of a clear stable reproduction method. The project contains more than a dozen different jobs and it's not easy to find which one is the exception. And it doesn't happen stably.

So would also like advice on what direction to debug from. 🙏

**Reply by agronholm** [reply]
The original poster didn't provide instructions either, so I'm at a loss as to how to debug this myself. I don't use APScheduler regularly myself, so I haven't encountered this problem myself, and it doesn't show up in the tests either. I need to be able to reproduce the issue somehow in order to do anything about it.
